src/Chapter2.hs

```diff
@@ -401,7 +401,7 @@ always matches (the same as a variable), but we don't use its value.
   the breakpoint, 'case' in Haskell always returns only a single
```

Comment:
```suggestion
  the breakpoint, _casing_ in Haskell always returns only a single
```

Reply:
Oh, I got what you mean, but is `casing` a good term? What you think about `pattern matching on function parameters`?

Reply:
Yeap, that is much better, thank you! 👍🏼 ❤️